### PR TITLE
data: Soham-glitch

### PR DESCRIPTION
**Pull request by Soham-glitch**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7508"></a>

### CODE REVIEW

File: data/Soham-glitch.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "Soham",
+    "type": "personal",
+    "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/Soham-glitch.json:4]: this is no longer necessary: 
```suggestion
```

File: data/Soham-glitch.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "bio": "Full Stack dev with a deep interest in Web 3, I am currently learning about blockchains and solidity, smart contracts, etc. I am also applying my full-stack skills to make websites. happy to collaborate",
+    "avatar": "https://github.com/Soham-glitch.png",
```
Comment (SaraJaoude) [inline data/Soham-glitch.json:6]: this is no longer necessary:
```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
I've deleted the sections in the json which are no longer needed (see my comments above) and this looks good to merge.

**Comment by Soham-glitch** [comment]
Thank you

**Pull request merged**